enhancements/baremetal/additional-ntp-servers.md

```diff
@@ -0,0 +1,104 @@
+
+A new field is required in install-config.yaml where the user can specify additional NTP servers that can then be used to set up a chrony config in the image customization of Ironic Python Agent and Cluster Baremetal Operator.
+
+## Motivation
+
+Users are trying to install Openshift clusters in environments without internet access and would like to be able to use custom internal NTP servers, as they don't want to manually update the time on each server. As stated above, if the time is more than 1 hour in the future the install will fail.
+
+### User Stories
+
+As a user, I want to specify custom NTP servers in install-config so that I can install a cluster without internet access.
+
+### Goals
+
+Have a dedicated install-config field where a user can specify a list of custom NTP servers that IPA will then sync to.
+
+`platform.baremetal.additionalNTPServers` - type: array, items: string, description: additionalNTPServers is a list of additional NTP servers to use for provisioning
+
+### Non-Goals
+
+## Proposal
+
+A new field in install-config will hold a list of NTP servers that will then be passed to the boostrap VM and from there to Ironic and Image Customization Controller. Or from install-config to the cluster and then Cluster Baremetal Operator and Ironic
+
+The propose field is `platform.baremetal.additionalNTPServers`. Description: additionalNTPServers is a list of additional NTP servers to use for provisioning. Type: array. Items: string. Values: List of hostnames or IP addresses.
+
+### Workflow Description
+
+- user inputs list of NTP servers
+- installer sets the environment variable for image-customization-controller and Ironic
+- the additional NTP servers are appended to the chrony config on masters
+- installer sets the key/value in the baremetal-provisioning-config for cluster-baremetal-operator
+- cluster-baremetal-operator configures the image customization
+- the additional NTP servers are appended to the chrony config on workers
+
+### API Extensions
+
+Not applicable
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+#### Standalone Clusters
+
+#### Single-node Deployments or MicroShift
+
+### Implementation Details/Notes/Constraints
+
+### Risks and Mitigations
+
+### Drawbacks
+
+## Open Questions [optional]
+
+## Test Plan
+
+A setup similar to what our custumers report: servers with the time set in the future by over one hour. With this setup and a custom NTP server the setup should complete succesfully.
+
+## Graduation Criteria
+
+Immediatly GA
+
+### Dev Preview -> Tech Preview
+
+### Tech Preview -> GA
+
+## Upgrade / Downgrade Strategy
+
+## Version Skew Strategy
+
+## Operational Aspects of API Extensions
+
+## Support Procedures
+
+## Alternatives
+
+None, this needs to be done in a way similar to what has been described in the proposal.
```

Comment:
The alternative is to have customers manually set the clock in BIOS, which is tedious and error-prone.